### PR TITLE
Update PluginRunner

### PR DESCRIPTION
**Pull request by mar-be**
Changes:
- New Python Script Artifacts 
- Use a DA instead of cloning the PluginRunner from GitHub

### CODE REVIEW

File: artifacttemplates/https%3A%2F%2Fust-quantil.github.io%2Fartifacttemplates/QHana-Plugin-Runner-Python_start-w1-wip1/ArtifactTemplate.tosca [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:ns0="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/artifacttemplates" id="artifactTemplatesQuantil-QHana-Plugin-Runner-Python_start-w1-wip1">
+    <ArtifactTemplate xmlns:otatyIgeneral="http://opentosca.org/artifacttypes" name="QHana-Plugin-Runner-Python_start-w1-wip1" type="otatyIgeneral:PythonScriptArtifact" id="QHana-Plugin-Runner-Python_start-w1-wip1">
```
Comment (lharzenetter) [inline artifacttemplates/https%3A%2F%2Fust-quantil.github.io%2Fartifacttemplates/QHana-Plugin-Runner-Python_start-w1-wip1/ArtifactTemplate.tosca]: Please also release this definition.

File: artifacttemplates/https%3A%2F%2Fust-quantil.github.io%2Fartifacttemplates/QHana-PluginRunner-Src-DA_w1-wip1/ArtifactTemplate.tosca [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:ns0="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/artifacttemplates" id="artifactTemplatesQuantil-QHana-PluginRunner-Src-DA_w1-wip1">
+    <ArtifactTemplate xmlns:ToscaBaseTypes="http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes" name="QHana-PluginRunner-Src-DA_w1-wip1" type="ToscaBaseTypes:ArchiveArtifact" id="QHana-PluginRunner-Src-DA_w1-wip1">
```
Comment (lharzenetter) [inline artifacttemplates/https%3A%2F%2Fust-quantil.github.io%2Fartifacttemplates/QHana-PluginRunner-Src-DA_w1-wip1/ArtifactTemplate.tosca]: s.a.

File: nodetypeimplementations/https%3A%2F%2Fust-quantil.github.io%2Fnodetypeimplementations/QHAna-PluginRunner-Implementation_latest-w2-wip1/LICENSE [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2021 University of Stuttgart
```
Comment (lharzenetter) [inline nodetypeimplementations/https%3A%2F%2Fust-quantil.github.io%2Fnodetypeimplementations/QHAna-PluginRunner-Implementation_latest-w2-wip1/LICENSE]: I guess the year is 2022.

File: nodetypeimplementations/https%3A%2F%2Fust-quantil.github.io%2Fnodetypeimplementations/QHAna-PluginRunner-Implementation_latest-w2-wip1/NodeTypeImplementation.tosca [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:ns0="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/nodetypeimplementations" id="nodeTypeImplementationsQuantil-QHAna-PluginRunner-Implementation_latest-w2-wip1">
+    <NodeTypeImplementation xmlns:nodeTypesQuantil="https://ust-quantil.github.io/nodetypes" targetNamespace="https://ust-quantil.github.io/nodetypeimplementations" name="QHAna-PluginRunner-Implementation_latest-w2-wip1" abstract="no" final="no" nodeType="nodeTypesQuantil:QHAna-PluginRunner_latest-w2-wip1">
```
Comment (lharzenetter) [inline nodetypeimplementations/https%3A%2F%2Fust-quantil.github.io%2Fnodetypeimplementations/QHAna-PluginRunner-Implementation_latest-w2-wip1/NodeTypeImplementation.tosca]: s.a.

However, release this as the last element as you need to update the references to the IA/DAs.

File: nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/QHAna-PluginRunner_latest-w2-wip1/LICENSE [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2021 University of Stuttgart
```
Comment (lharzenetter) [inline nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/QHAna-PluginRunner_latest-w2-wip1/LICENSE]: 2022.

File: nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/QHAna-PluginRunner_latest-w2-wip1/NodeType.tosca [hunk 6]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:ns0="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/nodetypes" id="nodeTypesQuantil-QHAna-PluginRunner_latest-w2-wip1">
+    <NodeType name="QHAna-PluginRunner_latest-w2-wip1" abstract="no" final="yes" targetNamespace="https://ust-quantil.github.io/nodetypes">
```
Comment (lharzenetter) [inline nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/QHAna-PluginRunner_latest-w2-wip1/NodeType.tosca]: s.a.

File: nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/QHAna-PluginRunner_latest-w2-wip1/README.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# QHAna  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)  
+
+> This node type installs and starts the QHAna plugin runner.
+
+## Properties
+
+- `MinWorkers. Minimum number of workers for the plugin runner
```
Comment (lharzenetter) [inline nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/QHAna-PluginRunner_latest-w2-wip1/README.md]: There is a backtick missing.

File: nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/QHAna-PluginRunner_latest-w2-wip1/README.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+- `MaxWorkers. Maximum number of workers for the plugin runner
```
Comment (lharzenetter) [inline nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/QHAna-PluginRunner_latest-w2-wip1/README.md]: There is a backtick missing.

File: nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/QHAna-PluginRunner_latest-w2-wip1/appearance/bigIcon.png.mimetype [hunk 9]
```diff
@@ -0,0 +1 @@
+image/png
```
Comment (lharzenetter) [inline nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/QHAna-PluginRunner_latest-w2-wip1/appearance/bigIcon.png.mimetype]: The image is quite large. can you create a smaller version with max. 100x100px?

**Review by lharzenetter — CHANGES_REQUESTED** [review]
Im currently running the deployment. However, some comments:

**Review by lharzenetter — APPROVED** [review]
Please check again, if the CORS issues are really gone before merging. Otherwise add the notice again.

**Comment by mar-be** [comment]
> Please check again, if the CORS issues are really gone before merging. Otherwise add the notice again.

CORS issues are resolved: https://github.com/UST-QuAntiL/tosca-definitions-qc-applications/pull/12/commits/c67a3ad956e1f9b27d33d725f633732253c58b04

**Comment by mar-be** [comment]
> Please check again, if the CORS issues are really gone before merging. Otherwise add the notice again.

https://github.com/UST-QuAntiL/tosca-definitions-qc-applications/pull/12 resolves the CORS issues

**Review by lharzenetter — APPROVED** [review]

**Comment by mar-be** [comment]
Depends on https://github.com/OpenTOSCA/tosca-definitions-common/pull/36

**Pull request merged**